tools/autograd/templates/python_linalg_functions.cpp

```diff
@@ -52,6 +52,22 @@ void initLinalgFunctions(PyObject* module) {
   if (PyModule_AddObject(module, "_linalg", linalg) != 0) {
     throw python_error();
   }
+
+  torch::linalg::Pytorch_LinAlgError = PyErr_NewException("torch.LinAlgError", PyExc_RuntimeError, nullptr);
```

Comment:
This can be moved to `torch/csrc/Exceptions.cpp` and be initialized at the same time as the other error (and bound to python the same way).
You can then bind it to `torch.linalg.LinAlgError` in the `__init__.py` of `torch/linalg`